backend/discord.py

```diff
@@ -2,22 +2,27 @@
 import httpx
 
 from backend.constants import (
-    OAUTH2_CLIENT_ID, OAUTH2_CLIENT_SECRET, OAUTH2_REDIRECT_URI
+    OAUTH2_CLIENT_ID, OAUTH2_CLIENT_SECRET
 )
 
 API_BASE_URL = "https://discord.com/api/v8"
 
 
-async def fetch_bearer_token(access_code: str) -> dict:
+async def fetch_bearer_token(code: str, redirect: str, *, refresh: bool) -> dict:
```

Comment:
Let's give `refresh` default value `False`.

Reply:
I don't think that really makes sense to be honest. It's a value that will highly depend on the situation, and will significantly alter the way the function works, so it should be explicitly specified by the caller.